certs/manager.go

```diff
@@ -498,7 +498,8 @@ func (c *CertificateManager) Add(certData []byte, orgID string) (string, error)
 	}
 	certID = orgID + certID
 
-	if cert, err := c.storage.GetKey("raw-" + certID); err == nil && cert != "" {
+	// use `Exist` rather than `Get` to avoiding infinite looping in mdcb storage
```

Comment:
The comment should be: "use `Exist` rather than `Get` to **avoid** infinite looping in MDCB storage"